### PR TITLE
BaSP-TG-162: shared profile form

### PR DESCRIPTION
**Pull request by ProDIGGY-vinilos**

### CODE REVIEW

File: src/Components/Pages/Admin/Employees/EmployeeForm/index.js [hunk 1]
```diff
@@ -1,159 +1,161 @@
...
-    setTypeModal('Success');
-  };
+  // const onSubmit = (data) => {
+  //   if (employeeId) {
+  //     dispatch(putEmployee(employeeId, data, token));
+  //   } else {
+  //     dispatch(postEmployee('', data));
+  //   }
+  //   setTypeModal('Success');
+  // };
```
Comment (fede230298) [inline src/Components/Pages/Admin/Employees/EmployeeForm/index.js:74]: Delete comment

File: src/Components/Pages/Admin/Employees/EmployeeForm/index.js [hunk 2]
```diff
@@ -1,159 +1,161 @@
...
+    //       />
+    //     </div>
+    //     <Button
+    //       onClick={handleSubmit(onSubmit)}
+    //       style="squaredPrimary"
+    //       disabled={false}
+    //       text="Save"
+    //     />
+    //   </form>
+    // </>
```
Comment (fede230298) [inline src/Components/Pages/Admin/Employees/EmployeeForm/index.js:158]: Delete comment

File: src/Components/Shared/ProfileForm/index.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+      openMessageModal();
+    }
+  }, [error, message]);
+
+  const reAuth = (data) => {
+    const user = auth.currentUser;
+    const credential = EmailAuthProvider.credential(user.email, data.password);
+
+    reauthenticateWithCredential(user, credential).catch((error) => {
+      console.log('error', error.message);
```
Comment (fede230298) [inline src/Components/Shared/ProfileForm/index.js:112]: Delete console.log

File: src/Components/Pages/Admin/Employees/EmployeeForm/index.js (hunk 1 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Pages/Admin/Employees/EmployeeForm/index.js]: Done

File: src/Components/Pages/Admin/Employees/EmployeeForm/index.js (hunk 2 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Pages/Admin/Employees/EmployeeForm/index.js]: Done

File: src/Components/Shared/ProfileForm/index.js (hunk 3 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Shared/ProfileForm/index.js]: Done

**Review by fede230298 — APPROVED** [review]

**Review by fede230298 — APPROVED** [review]

**Review by GonzaloAsencio — APPROVED** [review]

**Pull request merged**